System/Posix/Files.hsc

```diff
@@ -104,6 +104,12 @@ import System.Posix.Internals
 
 import Data.Time.Clock.POSIX (POSIXTime)
 
+-- throwErrnoTwoPathsIfMinus1_
+--
+-- | For operations that require two paths (e.g., renaming a file)
+throwErrnoTwoPathsIfMinus1_ loc path1 path2 =
+  throwErrnoIfMinus1_ (loc ++ " '" ++ path1 ++ "' to '" ++ path2 ++ "'")
```

Comment:
```suggestion
  throwErrnoIfMinus1_ (loc <> " '" <> path1 <> "' to '" <> path2 <> "'")
```
Just in case we ever move a `String` type that is not a `[Char]`, but something more like `UTF-8` text.  The more polymorphic `<>` should work even if `String` is no longer a list.  (Still holding out hope that could happen some day).